src/dashboard/src/components/accounts/views.py

```diff
@@ -62,6 +65,37 @@ def add(request):
     })
 
 
+def profile(request):
```

Comment:
This is a simplified version of the edit page which shows the user values as read only and just lets them regenerate their API key.